### DISCUSSION TITLE
Console

### DISCUSSION THREAD

**Discussion by dillingham** [spatie/ray#640] [Ideas]
Could be cool if you could select a directory and execute php from within ray.

Ray has so many good outputs that could be toggle-able beside the editor.

- [x] Show Queries
- [x] Show Events
- [x] Show Jobs
- [x] Show Http
- [ ] Show Cache

#### Code Editor: `~/Sites/Laravel-App`
```php
User::all()
```
[ Clear ] [ Execute ]

---

#### Code Editor: `~/Sites/Laravel-App`
```php
return new \App\Mail\NewUserMail();
```

[ Clear ] [ Execute ]

**Comment by freekmurze** [comment]
Could you elaborate on that idea?

**Reply by dillingham** [reply]
Sure, I updated the topic.
